apps/studio/components/interfaces/Database/Triggers/TriggerSheet.tsx

```diff
@@ -387,7 +387,7 @@ export const TriggerSheet = ({ selectedTrigger, open, setOpen }: TriggerSheetPro
                       <FormItemLayout layout="vertical" className="px-5">
                         <FormControl_Shadcn_>
                           <div className="flex flex-col gap-y-2">
-                            <p className="text-smn">Function to trigger</p>
+                            <p className="text-sm">Function to trigger</p>
```

Comment:
Just a typo I noticed elsewhere, unrelated to this effort. Feel free to remove if it muddies the later merge.